examples/pah_configure.yml

```
@@ -0,0 +1,95 @@
+---
+- name: Playbook to configure Private Automation Hub
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  collections:
+    - redhat_cop.ah_configuration
+
+  vars:
+    absent_present: "present"
+
+  pre_tasks:
+    - name: Include Vars
+      include_vars: vars/pah_vars.yml
```

This wont ever get included if including a tag.

```suggestion
      include_vars: vars/pah_vars.yml
      tags:
        - always
```